devops/helm/redis-cluster/values-0c27fb-dev.yaml

```diff
@@ -48,3 +45,9 @@ metrics:
   ## @param metrics.enabled Start a side-car prometheus exporter
   ##
   enabled: true
+  resources:
+    requests:
+      cpu: 50m
+      memory: 1024Mi
+    limits:
+      memory: 1024Mi
```

Comment:
Can we reduce the memory for this container? It's responsibility is just to provide the metrics sysdig, right?

Reply:
My bad for not suggesting some values in my previous comment.
Since this one is only for DEV right now and checking the metrics, we can go 200Mb or even less.
Since we have 6 redis pods also, the number here will also take 6x the configured value.
CPUwise, if our API has 10m as a configuration, I would suggest the same here. Does it make sense?